Tests/SwiftGenKitTests/StoryboardsMacOSTests.swift

```diff
@@ -64,4 +64,18 @@ class StoryboardsMacOSTests: XCTestCase {
 
     XCTDiffContexts(result, expected)
   }
+
+  func testAllStoryboardsWithCustomName() {
+    let parser = StoryboardParser()
+    do {
+      try parser.parseDirectory(at: Fixtures.directory(sub: .storyboardsMacOS))
+    } catch {
+      print("Error: \(error.localizedDescription)")
+    }
+
+    let result = parser.stencilContext(sceneEnumName: "XCTStoryboardsScene", segueEnumName: "XCTStoryboardsSegue")
```

Comment:
Mmmh maybe I misunderstood, where is that `stencilContext(sceneEnumName: segueEnumName: )` method defined, is it just a helper method for the tests here?